### ISSUE TITLE
Daily charge of battery

### ISSUE THREAD

**Issue by jais86** [klatremis/esphome-for-deye#41]
Hi,

Is it possible to add a daily charge and discharge of battery?

**Comment by fabhund** [comment]
No. To my knowledge there is no Modus register that has that value from the inverter itself.
Assuming you want this in HomeAssistant, then the energydashboard calculates this for you on a hourley/daily/weekly/monthly/quaterly/yearly basis anyway.

**Comment by Baduno** [comment]
What sensor do you use under "Energy - Home battery storage" to make this worke?

**Comment by luckylinux** [comment]
I'm not sure if it's being asked to CHARGE or DISCHARGE (like "Actually do it") or just what Amount of Charge/Discharge was done Today or in Total (like "Monitor").

For the latter, you can try
![image](https://github.com/user-attachments/assets/67b21fd3-6479-4bd7-a531-ead2fa433cec)

No guarantees, I did NOT test this

**Comment by luckylinux** [comment]
Actually I was using it for the "Total Charge/Discharge":
https://github.com/luckylinux/esphome-for-deye/blob/main/modules/battery.yaml

So you just would need to adapt that to registers 514 and 515 I guess.
